Source/santad/Logs/SNTEventLog.m

```diff
@@ -43,6 +44,9 @@ - (instancetype)init {
     _dateFormatter = [[NSDateFormatter alloc] init];
     _dateFormatter.dateFormat = @"yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
     _dateFormatter.timeZone = [NSTimeZone timeZoneWithName:@"UTC"];
+
+    // Grab the system UUID on init
+     _hostuuid = [SNTSystemInfo hardwareUUID];
```

Comment:
Use SNTConfigurator.machineID as that allows for the different methods of overriding the machineID (but falls back to this if none are set)